ncard_app/templates/detail_views/add_grant.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'events/base.html' %}
+{% load crispy_forms_tags %}
+{% load static %}
+<link rel="stylesheet" href="{% static 'style.css' %}" type="text/css" media="screen" >
+{% block title %}Add Grant{% endblock %}
+
+{% block content%}
+    <h1 class="ncard-dark-green">Add Grant</h1><br>
+            <div class="close_position"><a href="{% url 'list-grants' %}" class="aui_close">&times;</a></div>
```

Comment:
Inconsistent indentation

Reply:
Plesae ignore the other tables and forms for now, except for Awards

The rest I will update tomorrow once the this pull request has been approved